src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java

```diff
@@ -1636,4 +1640,12 @@ public static void deleteTargetBqmtTable() throws IOException, InterruptedExcept
       }
     }
   }
+
+  @After(order = 1, value = "@BQ_SECOND_RECORD_SOURCE_TEST")
```

Comment:
Should this be BQ_RECORD_SOURCE_TEST? I don't see this tag being used anywhere.

Reply:
we are using this is in bqtobq Additional.feature file. so **BQ_SECOND_RECORD_SOURCE_TEST** is correct

Reply:
I see, can we please rename both these tags to something meaningful?

It is hard to understand what these both tags are referring to `BQ_RECORD_SOURCE_TEST` & `BQ_SECOND_RECORD_SOURCE_TEST`?

Reply:
changed done.